examples/src/basic.d

```diff
@@ -43,6 +43,36 @@ class BasicPlugin : Plugin {
     event.msg.reply("Pong!");
   }
 
+  //An example command that clears messages in the channel
+  @Command("clear")
+  void onClearMessages(CommandEvent event){
```

Comment:
Nit; If you could make the bracket spacing line up with everything else (e.g. `) {` that would be awesome.